### ISSUE TITLE
TaskList missing CSS class

### ISSUE THREAD

**Issue by madskristensen** [lunet-io/markdig#14]
The GitHub flavor for tasks lists looks like this:

```md
- [ ] test
```

It renders in Markdig into:

```html
<li><input disabled="disabled" type="checkbox" /> test</li>
```

GitHub renders it with a class name on the `<li>` element so it can be styled differently so it doesn't have a CSS list style bullet

```html
<li class="task-list-item"><input disabled="disabled" type="checkbox" /> test</li>
```

And the `<ul>` gets a *contains-task-list* class attribute

**Comment by xoofx** [comment]
Should be fixed by commit 6a62ae9

**Comment by madskristensen** [comment]
Man this looks amazing now. Here's a screenshot:

![image](https://cloud.githubusercontent.com/assets/1258877/16182922/1dfdd400-3660-11e6-988e-4cf88b837bb5.png)

Thanks for adding this
